compute/cloud-client/src/test/java/compute/disks/DisksIT.java

```diff
@@ -265,13 +275,15 @@ public void afterEach() {
   }
 
   @Test
+  @Order(1)
```

Comment:
please consider to setup for each test separately instead of enforcing the order.
note that the order may fail upcoming tests despite that they are OK.

Reply:
Thank you. Deleted